packages/realm/src/Dictionary.ts

```diff
@@ -233,20 +233,40 @@ export class Dictionary<T = unknown> extends Collection<string, T, [string, T],
   }
 
   /**
- /**
-   * Add a key with a value or update value if key exists.
+   * Adds an element (or multiple ones) with the specified key and value to the dictionary or updates value if key exists.
+   * @param element The element to add
```

Comment:
```suggestion
   * Adds one or more elements with specified key and value to the dictionary or updates value if key exists.
   * @param element The object of element(s) to add
```

Another alternative is to incorporate "upserts" (popularly used about inserts or updates by the MDB drivers).